client/plots/profileBarchart.js

```diff
@@ -298,9 +298,10 @@ class profileBarchart extends profilePlot {
 	}
 }
 
-export async function getPlotConfig(opts, app) {
+export async function getPlotConfig(opts, app, _activeCohort) {
 	try {
-		const defaults = getProfilePlotConfig(app, opts)
+		const activeCohort = _activeCohort === undefined ? app.getState().activeCohort : _activeCohort
```

Comment:
please document why the 3rd param `_activeCohort` is needed and it can't be derived from app.getState().activeCohort?

Reply:
When recovering a session getState() returns null and that is why we need to pass the activeCohort.